webdriver/tests/bidi/browsing_context/event_context_created.py

```diff
@@ -0,0 +1,24 @@
+import pytest
```

Comment:
I'm currently working on the implementation for this event in Firefox. So if you don't mind I can cover the tests here in more detail.

Reply:
would be great, thanks!

Reply:
@sadym-chromium is it ok with you when I'm going to use some APIs from WebDriver HTTP for now to properly test this event? I'm thinking of `New Window`, `Close`, `Navigate To` and `Execute Script`. The latter we would need because we don't have script execution implemented yet and we could call `window.open()`. We would update to BiDi commands as soon as appropiate commands are implemented.

Reply:
An initial set of tests for `contextCreated` has been landed via PR #32829. These should be further extended when we have broader coverage.